wallet/invoices.c

```diff
@@ -61,8 +61,10 @@ static bool wallet_stmt2invoice(sqlite3_stmt *stmt, struct invoice *inv)
 	/* Correctly 0 if pay_index is NULL. */
 	inv->pay_index = sqlite3_column_int64(stmt, 7);
 
-	if (inv->state == PAID)
+	if (inv->state == PAID) {
 		inv->msatoshi_received = sqlite3_column_int64(stmt, 8);
+		inv->paid_timestamp = sqlite3_column_int64(stmt, 9);
```

Comment:
Sort of uses implicit NULL -> 0 conversion, should be fine though.

Reply:
`paid_timestamp` should get set at the point that `state` becomes `PAID`, xref `msatoshi_received`, which is similar.

We are actually slightly inconsistent here, since `pay_index` is `NULL` on-db if `state != PAID` too, and yet we always load it.